lib/splitclient-rb/engine/api/client.rb

```diff
@@ -6,7 +6,8 @@ class Client
       RUBY_ENCODING = '1.9'.respond_to?(:force_encoding)
 
       def get_api(url, config, api_key, params = {})
-        api_client.get(url, params) do |req|
+        parsed_url = URI.parse(url)
```

Comment:
what is this for max?

Reply:
Long explanation is available from the link above. Short: I had an issue when I've seen phrase
`no_proxy is not supported` flooding logs (it appeared one time for 1 request to Split API), this is the solution.